test/test_config.dart

```diff
@@ -4,15 +4,12 @@ import 'package:draw/src/draw_config_context.dart';
 void main() {
   test('Simple tests for default section of local file.', () {
     final DRAWConfigContext configContext = new DRAWConfigContext();
-    expect(configContext.shortUrl, equals(Uri.parse('https://redd.it')));
-    expect(
-        configContext.redditUrl, equals(Uri.parse('https://www.reddit.com')));
     expect(configContext.clientId, equals('Y4PJOclpDQy3xZ'));
     expect(configContext.clientSecret, equals('UkGLTe6oqsMk5nHCJTHLrwgvHpr'));
     expect(configContext.password, equals('pni9ubeht4wd50gk'));
     expect(configContext.username, equals('fakebot1'));
   });
-
+/*
```

Comment:
Why are these commented out? We do need more test coverage of this for simple and complex cases and malformed config files.

Reply:
Yeah, sorry about that, I'll add back the comprehensive tests. I had deleted the corresponding ini file, so this would have blocked me from testing out all the other additions made in this branch. :)